packages/apps/src/microsoft/teams/apps/plugins/metadata.py

```diff
@@ -20,10 +20,15 @@ class PluginOptions:
     description: Optional[str] = None
 
 
-def Plugin(name: Optional[str] = None, version: Optional[str] = None, description: Optional[str] = None) -> Any:
+T = TypeVar("T")
+
+
+def Plugin(
+    name: Optional[str] = None, version: Optional[str] = None, description: Optional[str] = None
+) -> Callable[[Type[T]], Type[T]]:
     """Turns any class into a plugin using the decorator pattern."""
 
-    def decorator(cls: Type[Any]) -> Any:
+    def decorator(cls: Type[T]) -> Type[T]:
```

Comment:
Had to use generics here because without it, this definition was setting the class to be "Any" which was losing all its type checking goodness.